src/data.js

```diff
@@ -0,0 +1,83 @@
+import './style.css';
+import { error, weatherReport } from './app';
+
+const buttonDiv = document.querySelector('.button-div');
+buttonDiv.style.display = 'none';
+
+const bgOne = document.querySelector('.body-one');
+
+const changeBackground = (temp) => {
+  if (temp.description === 'cloudy') {
+    bgOne.style.backgroundImage = 'url(./images/cloud.gif)';
+  } else if (temp.description === 'rain') {
+    bgOne.style.backgroundImage = 'url(./images/rain.gif)';
+  } else if (temp.description === 'sun') {
+    bgOne.style.backgroundImage = 'url(./images/sunligth.jpg)';
+  } else if (temp.description === 'thunder') {
+    bgOne.style.backgroundImage = 'url(./images/thunder.gif)';
+  } else if (temp.description === 'fog') {
+    bgOne.style.backgroundImage = 'url(./images/fog.gif)';
+  } else {
+    bgOne.style.backgroundImage = 'url(./images/snow.gif)';
+  }
+};
```

Comment:
:bulb: I've noticed that you declared a function to alter the image with a different temperature description from the API but then you linked the images wrongly for your production code. 
:thinking:  Also, keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)